File: 1_S/csharp/Kullanici.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Eticarett1_
{
    public class Kullanici
    {
        private string isim;
        private string mail;
        private Adres adres;

        public void SetIsim(string isim)
        {
            this.isim = isim;
        }

        public string GetIsim()
        {
            return isim;
        }

        public string GetMail()
        {
            return mail;
        }

        public void SetMail(string mail)
        {
            this.mail = mail;
        }

        public void AdresGuncelle(Adres adres)
        {
            this.adres = adres;
        }

        public string GetAdres()
        {
            return adres.ToString();
        }

        public override string ToString()
        {
            return GetIsim() + " " + GetMail() + " " + GetAdres();
        }

        public void GirisYap(string mail, string sifre)
        {
            System.Console.WriteLine("Giriş yapıldı.");
        }
    }

}

File: 1_S/java/src/Kullanici.java
public class Kullanici{
    private String isim,mail;
    private Adres adres;


/*
    public void adresGuncelle(Adres adres){
        this.adres.setIl(adres.getIl());
        this.adres.setIlce(adres.getIlce());
        this.adres.setUlke(adres.getUlke());
        this.adres.setPostKodu(adres.getPostKodu());
        
        //this.adres=adres;
    }
*/


    public void setIsım(String isim){
        this.isim=isim;
    }

    public void setMail(String mail){
        this.mail=mail;
    }
    
    public String getIsim(){
        return isim;
    }
    public String getMail(){
        return mail;
    }


    
    public void adresGuncelle(Adres adres) {
        this.adres = adres;
    }

    public String getAdres() {
        return adres.toString();
    }

    @Override
    public String toString() {
        return getIsim() + " " + getMail() + " " + getAdres();
    }
    public void girisYap(String mail, String sifre) {
        System.out.println("Giriş yapıldı.");
    }

}